public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -15,6 +24,7 @@ import { CreateTokenModal, ServiceAccountToken } from './components/CreateTokenM
 import ServiceAccountListItem from './components/ServiceAccountsListItem';
 import {
   changeQuery,
+  changePage,
```

Comment:
Oh, wait, pagination functionality was there it seems :)

Reply:
Yep it was. I was surprised we had all the logic there but not using it